README.md

```diff
@@ -29,8 +29,7 @@ Please see [Environment Support](https://firebase.google.com/support/guides/envi
 #### Node.js
 
 Before you can start working on the Firebase JS SDK, you need to have Node.js
-installed on your machine. As of June 13th, 2023 the team has been testing with Node.js versions
-between `16.4.0` and `16.16.0`, but the required verison of Node.js may change as we update
+installed on your machine. As of April 19th, 2024 the team has been testing with Node.js version 20.12.2, but the required verison of Node.js may change as we update
```

Comment:
nit, can you put the version in `backticks`, and keep the text wrapping?

Reply:
Oops- ty for catching! I've updated the commit 👍